src/constants/test-android.js

```diff
@@ -107,7 +107,7 @@ export const testAndroidDataConstants = [
         load: 'metadataLoad',
     },
     {
-        title: 'Data download size',
+        title: 'Data download size   (KB)',
```

Comment:
why are there multiple spaces here?  This should also be translated, but that could be a separate PR

Reply:
I'm cleaning the extra spaces, the translation will be in other PR